db/fixtures/miq_shortcuts.yml

```diff
@@ -26,7 +26,7 @@
   :startup: true
 - :name: chargeback_assignments
   :description: Overview / Chargeback / Assignments
-  :url: /chargeback_assignments/explorer
+  :url: /chargeback_assignments/index
```

Comment:
You don't need index, just remove the last `/` and everything after and it should work.